x-pack/test/functional/apps/observability_logs_explorer/alerts.ts

```diff
@@ -30,7 +30,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       expect(toastTitle).toMatch(/created/i);
     });
 
-    it('should create SLO successfully', async () => {
+    // Flaky test
+    it.skip('should create SLO successfully', async () => {
```

Comment:
**question:** why is this test marked as flaky? Does it make sense to keep it implemented if will be disabled from the beginning?

Reply:
There's an intermittent issue where the good query field does not get populated which stops the form from getting submitted

Reply:
Is there anything we can do about this case? Having a flaky test disabled since the beginning doesn't bring additional value, you may consider directly removing it if it's not possible to stabilize it.